docs/product/issues/issue-details/index.mdx

```diff
@@ -102,6 +102,36 @@ If you’ve enabled [Session Replay](/product/explore/session-replay/), you’ll
 
 ![Session Replay](./img/issue-replay.png)
 
+## Feature Flags
+There are two types of integrations you can configure to integrate feature flag data into your error events.
+
+The first type is flag evaluation tracking within the Sentry SDK. To enable it, update the Sentry SDK's "init" method to include one of our supported feature flag integrations. Support varies by language and by provider. Doing so will give you access to the feature flag table which shows any flags that were evaluated after the Sentry SDK's initialization and before the error event occurred. You will see rows of flag, evaluation-result pairs.
+
+The second type is feature flag change tracking. This will enable a timeline of feature flag definition changes inside the releases graph. These annotations can help you diagnose when a feature flag change is related to a new issue. To enable this type of integration, please visit the documentation page of one of our supported providers:
+
+* [LaunchDarkly](/organization/integrations/feature-flag/launchdarkly/#change-tracking-integration)
+
+When both integrations are enabled, you'll also see suspect feature flags highlighted in yellow in the feature flag table. Feature flags are marked as "suspect" if they're modified suspiciously close to the detection of a new issue.
+
+![Feature Flags](./img/feature-flags.png)
+
+A timeline of feature flag changes is displayed in the releases graph. It can help you diagnose when a feature flag change has triggered a new bug report.
```

Comment:
```suggestion
A timeline of feature flag changes is displayed in the event and user volume chart. It can help you diagnose when a feature flag change has triggered a new error.
```